modules/ROOT/pages/comments-commands-events-apis.adoc

```diff
@@ -67,3 +67,63 @@ console.log(comments.getEventLog(
   { after: '2022-02-22T12:34:56Z' }  // ISO-8601 standard: YYYY-MM-DDThh:mm:ssZ
 ));
 ----
+
+[[editorannotator]]
+=== `+editor.annotator+`: `+'tinycomments'+` annotator
```

Comment:
The other API documented on this page is `getEventLog()`, and that is the string used for the sub-head that start’s the docs for that API.

I’m not arguing against this second sub-head, per se. But is there an edit that would make the new sub-head present more in line with the extant sub-head?